src/plugins/ssh/index.ts

```diff
@@ -146,6 +145,36 @@ const accessPropagationGuard = (
   };
 };
 
+/**
+ * Parses and prints a chunk of SSH output to stderr.
+ *
+ * If debug is enabled, all output is printed. Otherwise, only selected messages are printed.
+ *
+ * @param chunkString the chunk to print
+ * @param debug true if debug output is enabled
+ */
+const parseAndPrintSshOutputToStderr = (
+  chunkString: string,
+  debug?: boolean
+) => {
+  const lines = chunkString.split("\n");
+
+  // SSH only prints the "Authenticated to" message if the verbose option (-v) is enabled
```

Comment:
So the user has to pass --debug in order to get the `Authenticated to` message?
Then do we have to always pass -v to ssh even the user didn't pass --debug? 
Do we already force -v somewhere (since it's not in this PR)?

Reply:
It's here in this PR. I'll make the message more clear.